packages/compass-import-export/src/modules/export.js

```diff
@@ -277,11 +284,12 @@ export const queryChanged = (query) => ({
  * @param {Number} document count given current query.
  * @param {Object} current query.
  */
-export const onModalOpen = ({ namespace, count, query }) => ({
+export const onModalOpen = ({ namespace, count, query, aggregation }) => ({
   type: ON_MODAL_OPEN,
   namespace,
   count,
-  query,
+  query: query || {},
```

Comment:
Same as for aggregation, empty object is not a great default / empty value due to ambiguity